tests/test_bounds.sh

```diff
@@ -3,17 +3,18 @@
 set -e
 
 : "${SSHPORT:=10021}"
-RESULT="fail"
 if [ -f "../bin/bounds" ]; then
     scp -P $SSHPORT ../bin/bounds root@127.0.0.1:/root
     RESULT={$(ssh -p $SSHPORT -t root@127.0.0.1 'sh -c "/root/bounds 2>&1 > /dev/null"') && exit 1} || true
-    if [ -z "${RESULT##*In-address space security exception (core dumped)*}" ]; then
-        echo "bounds... ok"
+    echo -n "bounds... "
+    exit_status=0
```

Comment:
I don't think this has been tested? This line sets exit_status then the next line immediately reads the same variable, so nothing can have changed inbetween -- and the `ssh` line still has `exit 1` in it.

Reply:
I've just found this. That's a mix of stuff that's why you see two commits. I'll address and push.